index.html

```diff
@@ -181,6 +181,69 @@ <h5>Languages</h5>
           </div>
         </section>
       </section>
+
+      <section class="section-contact">
+        <div class="container">
+          <p class="contact-description">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </p>
+          <form
+            class="form"
+            action="https://formspree.io/f/xpzewdnr"
+            method="post"
+          >
+            <div class="form-control">
+              <input required type="text" id="name" placeholder="Full name" />
+            </div>
+            <div class="form-control">
+              <input required type="email" placeholder="Email address" />
+            </div>
+            <div class="form-control">
+              <textarea
+                required
+                id=""
+                cols="30"
+                rows="6"
+                maxlength="500"
+                placeholder="Hey I'd like to connect to ..."
+              ></textarea>
+            </div>
+            <button class="btn">Get in touch</button>
+          </form>
```

Comment:
Kindly give the pressed state of your form a styling as stated in the Figma design. 😊 

<img width="855" alt="image" src="https://user-images.githubusercontent.com/73854999/228647796-70eecc53-cbd4-4f20-ab27-2fe7da5918f0.png">

Reply:
@Lizdev-05  thanks for reviewer 
there a small question the pressed state if visited right